### PR TITLE
Add 5.20.2 release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release notes for 5.20.2

## Motivation and Context
Support patch release 5.20.2

### CODE REVIEW

File: content/sensu-go/5.20/release-notes.md [hunk 1]
```diff
@@ -61,6 +62,27 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+See the [upgrade guide][1] to upgrade Sensu to version 5.20.2.
+
+**NEW FEATURES:**
+
+- The API request log now includes the username.
+
+**FIXES:**
+
+- ([Commercial feature][141]) Process discovery in the agent is temporarily disabled.
+The [`processes` field ][143] now contains only the name of the process.
```
Comment (hillaryfraley) [inline content/sensu-go/5.20/release-notes.md:80]:
```suggestion
- ([Commercial feature][141]) Process discovery in the agent is temporarily disabled.
```

Comment (hillaryfraley) [inline content/sensu-go/5.20/release-notes.md:80]:
```suggestion
- ([Commercial feature][141]) Process discovery in the agent is temporarily disabled.
```

File: content/sensu-go/5.20/release-notes.md [hunk 2]
```diff
@@ -61,6 +62,26 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.20.2 release notes
+
+**May 26, 2020** &mdash; The latest release of Sensu Go, version 5.20.2, is now available for download.
+
+This patch release adds username to the API request log to help operators with troubleshooting and user activity reporting, as well as validation for subjects in role-based access control (RBAC) role binding and cluster role binding. Release 5.20.2 also temporariliy disables process discovery so we can investigate and resolve an increase in event payload size.
```
Comment (palourde) [inline content/sensu-go/5.20/release-notes.md]: > so we can investigate and resolve an increase in event payload size.

Maybe we should instead mention the performance impact of this feature on the backend

Comment (hillaryfraley) [inline content/sensu-go/5.20/release-notes.md]:
```suggestion
This patch release adds username to the API request log to help operators with troubleshooting and user activity reporting, as well as validation for subjects in role-based access control (RBAC) role binding and cluster role binding.
Release 5.20.2 also temporarily disables process discovery so we can investigate and resolve its performance impact on the backend (increased CPU and memory usage).
```

Comment (hillaryfraley) [reply content/sensu-go/5.20/release-notes.md]: 👍 what do you think of this change?
```
Release 5.20.2 also temporarily disables process discovery so we can investigate and resolve its performance impact on the backend (increased CPU and memory usage).
```

**Review by palourde — APPROVED** [review]
:shipit:

**Pull request merged**